### PR TITLE
add dubbo + spring boot + mybatis & hibernate samples

### PR DESCRIPTION
**Pull request by francisoyc**
add dubbo + spring boot + mybatis & hibernate samples and remove old spring xml samples，all samples have run successfully

### CODE REVIEW

File: 99-integration/dubbo-samples-hibernate/case-versions.conf [hunk 1]
```diff
@@ -1,25 +0,0 @@
...
-#   limitations under the License.
-#
-
-
-# Supported component versions of the test case
-
-# Spring app
-#dubbo.version=2.7.*, 3.*
-dubbo.version=4.*
-spring.version=4.*
```
Comment (AlbumenJ) [inline 99-integration/dubbo-samples-hibernate/case-versions.conf:25]: Why remove this file

File: 99-integration/dubbo-samples-hibernate/dubbo-samples-spring-boot-hibernate-api/pom.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```
Comment (AlbumenJ) [inline 99-integration/dubbo-samples-hibernate/dubbo-samples-spring-boot-hibernate-api/pom.xml:2]: Please add ASF license header

File: 99-integration/dubbo-samples-hibernate/dubbo-samples-spring-boot-hibernate-consumer/pom.xml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```
Comment (AlbumenJ) [inline 99-integration/dubbo-samples-hibernate/dubbo-samples-spring-boot-hibernate-consumer/pom.xml:2]: ASF license header required

**Review by AlbumenJ — CHANGES_REQUESTED** [review]

**Comment by AlbumenJ** [comment]
The missing license header check can be found in https://github.com/apache/dubbo-samples/actions/runs/3739122037/jobs/6351265714

File: 99-integration/dubbo-samples-hibernate/case-versions.conf (hunk 1 above)
Comment (francisoyc) [reply 99-integration/dubbo-samples-hibernate/case-versions.conf:25]: I will restore it back

File: 99-integration/dubbo-samples-hibernate/dubbo-samples-spring-boot-hibernate-api/pom.xml (hunk 2 above)
Comment (francisoyc) [reply 99-integration/dubbo-samples-hibernate/dubbo-samples-spring-boot-hibernate-api/pom.xml:2]: ok

File: 99-integration/dubbo-samples-hibernate/dubbo-samples-spring-boot-hibernate-consumer/pom.xml (hunk 3 above)
Comment (francisoyc) [reply 99-integration/dubbo-samples-hibernate/dubbo-samples-spring-boot-hibernate-consumer/pom.xml:2]: ok

**Review by AlbumenJ — CHANGES_REQUESTED** [review]
Please fix conflicts. Others LGTM

**Comment by AlbumenJ** [comment]
Please fix test cases

**Pull request merged**